src/Maestro/MaestroApplication/ApplicationPackageRoot/ApplicationManifest.xml

```diff
@@ -132,6 +132,6 @@
     <DefaultRunAsPolicy UserRef="MaestroUser" />
   </Policies>
   <Certificates>
-    <EndpointCertificate X509FindType="FindBySubjectName" X509FindValue="[WebCertName]" Name="WebSslCert" />
```

Comment:
I think this will break things. I don't know why VS is trying to change it, but the default isn't "FindBySubjectName" so this will break the cert finding.

Reply:
Weird. I'll revert this. I didn't even touch this code... VS is just being VS.